salt/cloud/clouds/profitbricks.py

```diff
@@ -287,29 +288,38 @@ def get_datacenter(conn):
     )
 
 
-def get_image(conn, vm_):
+def get_disk_type(vm_):
     '''
-    Return the image object to use
+    Return the type of disk to use. Either 'HDD' (default) or 'SSD'.
     '''
-    vm_image = config.get_cloud_config_value('image', vm_, __opts__).encode(
-        'ascii', 'salt-cloud-force-ascii'
+    return config.get_cloud_config_value(
+        'disk_type', vm_, __opts__, default='HDD',
+        search_global=False
     )
 
-    for item in conn.list_images()['items']:
-        img = {'id': item['id'], 'name': item['properties']['name']}
 
-        if isinstance(img['id'], string_types):
-            img_id = img['id'].encode('ascii', 'salt-cloud-force-ascii')
-        else:
-            img_id = str(img['id'])
+def get_wait_timeout(vm_):
+    '''
+    Return the wait_for_timeout for resource provisioning.
+    '''
+    return config.get_cloud_config_value(
+        'wait_for_timeout', vm_, __opts__, default=15 * 60,
+        search_global=False
+    )
 
-        if isinstance(img['name'], string_types):
-            img_name = img['name'].encode('ascii', 'salt-cloud-force-ascii')
-        else:
-            img_name = str(img['name'])
 
-        if vm_image and vm_image in (img_id, img_name):
-            return img
+def get_image(vm_):
```

Comment:
This looks like it went from two arguments to one. Is this going to be a problem?